sklearn/metrics/tests/test_ranking.py

```diff
@@ -467,7 +467,7 @@ def test_precision_recall_curve_pos_label():
     assert_array_almost_equal(r, r2)
     assert_array_almost_equal(thresholds, thresholds2)
     assert_equal(p.size, r.size)
-    assert_equal(p.size, thresholds.size + 1)
```

Comment:
Ah, this is an issue due to the appendings in the return statement.   When  `last_ind == thresholds.size - 1` (which occurs when the smallest `y_scores` value has ground truth `True`), there are no more threshold values available to make the returned `thresholds` array have the same size as `precision` and `recall`.

If we want to keep array sizes consistent, in this case we could simply double the final threshold value:
`np.r_[thresholds, thresholds[-1]]`

If we want to keep sizes consistent